ui/components/app/create-new-vault/create-new-vault.js

```diff
@@ -126,7 +130,7 @@ export default function CreateNewVault({
       </div>
       {includeTerms ? (
         <div className="create-new-vault__terms">
-          <CheckBox
+          <Checkbox
```

Comment:
Can we use the `label` prop that comes with the Checkbox rather than the `<label>` JSX?

Reply:
Sure,
will do that now.